nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -119,6 +119,8 @@
 
 - `linuxPackages.nvidiaPackages.dc_520` has been removed since it is marked broken and there are better newer alternatives.
 
+- `programs.less.lessopen` set default preprocessor to null.
```

Comment:
```suggestion
- `programs.less.lessopen` is now null by default. To restore the previous behaviour, set it to `''${lib.getBin pkgs.lesspipe "lesspipe.sh"} %s''`.
```

Reply:
Thanks @FliegendeWurst, Ive incorporated that into the commit

Reply:
Oh I mistyped. It should be lib.getExe' not lib.getBin